public/app/features/explore/Explore.test.tsx

```diff
@@ -113,6 +113,11 @@ jest.mock('app/core/core', () => ({
   },
 }));
 
+jest.mock('@grafana/runtime', () => ({
+  ...jest.requireActual('@grafana/runtime'),
+  getPluginLinkExtensions: () => ({ extensions: [] }),
+}));
```

Comment:
Should we add a basic test for checking if the extension point renders extensions if they are registered?